Plots/XY/NCL_xy_3.py

```diff
@@ -0,0 +1,78 @@
+"""
+NCL_xy_3.py
+===============
+   - Reversing the Y axis
+   - Changing the line dash pattern in an XY plot
+   - Creating your own line dash pattern for an XY plot
+   - Changing the line color and thickness in an XY plot
+   - Creating a vertical profile plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/xy_3.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/xy_3_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/xy_3_2_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+ds = ds.U
+ds = ds.isel(time=0).drop('time')
+ds = ds.isel(lon=0).drop('lon')
+ds = ds.isel(lat=42).drop('lat')
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(8, 8))
+ax = plt.gca()
+
+# Plot data
+plt.plot(ds.data, ds.lev)
```

Comment:
Can you define line color here as you did in line 66?

Reply:
I refrained from doing that because the NCL example first plots a line with the default characteristics. NCL and matplotlib have different defaults (color and linestyle being two of them) so I wanted to showcase matplotlib's defaults of a solid, blue line here. Do you think that's appropriate for this example?

Reply:
I think that's fair. I wanted to make sure this was addressed in case it was an issue within any of the functions and would need to be fixed